src/crc32.c

```diff
@@ -31,7 +31,7 @@
 #include "crc32.h"
 
 #include <trivia/config.h>
-#include <third_party/crc32.h>
+#include <crc32_impl.h>
```

Comment:
I believe we'll have crc32_impl.h and crc32.c in third_party folder after rename only first one?
Is it consistent? For me it looks natural if header has the same name as the code file.

Reply:
Fixed